grails-app/controllers/org/pih/warehouse/requisition/RequisitionTemplateController.groovy

```diff
@@ -460,27 +460,4 @@ class RequisitionTemplateController {
         redirect(action: "batch", id: params.id)
 
     }
-
-    private List<Location> getDepots() {
```

Comment:
@jmiranda the same two methods are on the `RequisitionController`, and seem to be not used too. Perhaps removing those would be useful too?